pkg/process/config/yaml_config.go

```diff
@@ -61,6 +62,10 @@ func (a *AgentConfig) loadSysProbeYamlConfig(path string) error {
 	}
 
 	a.SysProbeBPFDebug = config.Datadog.GetBool(key(spNS, "bpf_debug"))
+	if config.Datadog.IsSet(key(spNS, "bpf_dir")) {
+		bytecode.DefaultBPFDir = config.Datadog.GetString(key(spNS, "bpf_dir"))
+	}
```

Comment:
This is inconsistent with how config is managed through the rest of the system probe. Usually, we add a field on `config` and pass it in where it's needed.

Reply:
True. I did try this approach first. But it implies passing this configuration setting in many places : tracer, oom, tcp queue length and security agent. I can go this way though